packages/poi-preset-karma/index.js

```diff
@@ -59,16 +59,26 @@ module.exports = (options = {}) => {
       files = ensureArray(files)
       files.push({ pattern: 'static/**/*', watched: false, included: false, served: true, nocache: false })
 
+      // TODO use webpack-chain to look up?
+      const isTypeScript = webpackConfig.module.rules
+        .map(r => r.use)
+        .reduce((acc, u) => acc.concat(u), [])
+        .map(u => u.loader)
+        .some(l => l === 'ts-loader')
```

Comment:
Do you have to wrap everything in poi.extendWebpackConfig in order to get access to the chain?

(I know this approach is stupid, but wanted to focus on the actual PR, hehe)

Reply:
`poi.extendWebpackConfig` will be ran before `poi.run`, so you can check if typescript rule exists using webpack-chain in that scope and use it in `poi.run`

Reply:
Okay thanks. Will try that

Reply:
@egoist changed it as you suggested